File: Python/isPrime.py
#!/usr/bin/env python3
import sys
import math

num = int(sys.argv[1])
''' comment'''


def prime_test(n):
    if n == 2:
        return ("2 is prime number!")
    for i in range(2, n - 1):
        if (n % i == 0):
            return ("%i is not prime number." % (n))
        elif i >= math.sqrt(n):
            return ("%i is prime number!" % (n))


print(prime_test(num))

File: javascript/deno-static-execute-binary/is-prime.ts
export const isPrimeNumber = (v: number): boolean => {
  if (v === 1) {
    return false;
  }

  for (let i = 2; i < Math.sqrt(v); i++) {
    if (v % i === 0) {
      return false;
    }
  }
  return true;
}
